packages/client/tests/functional/relationMode/tests_m-to-n.ts

```diff
@@ -1047,23 +1061,7 @@ testMatrix.setupTestSuite(
             )
           })
 
-          // For all databases (PostgreSQL, SQLite, MySQL, SQL Server, CockroachDB & MongoDB)
-          // onDelete: SetNull & relationMode: prisma
-          // fails the 2 following tests
-          // they are a copy above the tests above but with relationMode: prisma and `.failing`
-          // So we can run all the tests successfully
-          //
-          // For the first test:
-          // Received promise resolved instead of rejected
-          // Resolved to value: {"id": "1", "published": null}
-          //
-          // For the second test:
-          // Received promise resolved instead of rejected
-          // Resolved to value: {"id": "1-cat-a", "published": null}
-          //
-          // See issue https://github.com/prisma/prisma/issues/15683
-
-          testIf(isRelationMode_prismaAndSetNull).failing(
+          testIf(isRelationMode_prismaAndSetNull)(
```

Comment:
Do we still need these duplicated tests? or can we remove them if they error like the test with same name but `!isRelationMode_prismaAndSetNull` condition?

Reply:
That should be done now.